go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20
 	github.com/moby/moby v27.1.1+incompatible
 	github.com/moby/term v0.5.0
-	github.com/ninech/apis v0.0.0-20240918081833-57655d99d868
+	github.com/ninech/apis v0.0.0-20241011133500-fc6e5a3bd98c
```

Comment:
Reminder to myself to change this back to main before merging as this depends on a yet to be merged API type change.